build/miengine.targets

```diff
@@ -47,6 +47,22 @@
     </ItemGroup>
   </Target>
 
+  <!-- Custom target that will be invoked after signing and loc to copy signed satellite assemblies
+       to the drop location.
+       This target is conditioned to only run if localization is currently turned on. -->
+  <Target Name="CopySatellitesToDrop" AfterTargets="SignFiles" Condition="'$(LocalizationEnabled)'=='true'">
```

Comment:
You used LocalizedOutputs in the other target but you use CopySatellites here. Should this be CopySignedLocalizedOutputsToDrop?

Also, I feel like this should be "AfterTargets='GatherLocalizedOutputsForSigning'"

Reply:
Maybe, I thought something like copy signed localized outputs was getting a bit lengthy, so I tried to find a more concise name that was still clear. 
I think after GatherLocalizedOutputsForSigning is too early because the signing hasn't happened yet.

Reply:
fair enough. the naming didn't seem to link the two together which is why I commented.